genesis/src/genesis_accounts.rs

```diff
@@ -1,27 +1,471 @@
-use solana_sdk::{account::Account, pubkey::Pubkey, system_program};
+use crate::{lockups::Lockups, stakes::create_stakes};
+use hex;
+use solana_sdk::{
+    account::Account, genesis_config::GenesisConfig, native_token::sol_to_lamports, pubkey::Pubkey,
+    system_program,
+};
 
-pub(crate) fn create_genesis_accounts(
+pub(crate) fn add_genesis_accounts(
+    genesis_config: &mut GenesisConfig,
     mint_pubkey: &Pubkey,
-    mint_lamports: u64,
-) -> Vec<(Pubkey, Account)> {
-    vec![
+    mut mint_lamports: u64,
+) {
+    // Lockups for the first set of genesis stakers
+    let lockups = Lockups::by_years(
+        0.2,
+        0.5,
+        24,
+        1.0 / 12.0,
+        &genesis_config.epoch_schedule,
+        &genesis_config.poh_config.target_tick_duration,
+        genesis_config.ticks_per_slot,
+    );
+    [
+        (
```

Comment:
closest thing to human readable I could get, thoughts?

Reply:
Since they're already on separate lines, how about a struct constructor for each item? It'd read like JSON.